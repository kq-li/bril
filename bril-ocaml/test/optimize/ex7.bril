@main {
  x: int = const 1;
.lbl1:
.lbl2:
  y: int = const 2;
  jmp .lbl1;
.lbl3:
}
