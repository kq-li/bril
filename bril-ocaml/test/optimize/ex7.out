@main {
.lbl1:
.lbl2:
  jmp .lbl1;
.lbl3:
}
